cases/rescript-schema.ts

```diff
@@ -2,8 +2,24 @@ import * as S from 'rescript-schema';
 
 import { createCase } from '../benchmarks';
 
-createCase('rescript-schema', 'parseSafe', () => {
-  const schema = S.object({
+// To bypass the "Assertions require every name in the call target to be declared with an explicit type annotation.ts(2775)"
+// when using assert
+type Data = {
```

Comment:
Can you please clarify this requirement?

Re-declaring types go against the ideals of this test repository.

Reply:
It's a TS thing. I've managed to work around it by adding `!` after `assert`. The `effect-schema` lib uses the same approach, so probably it's fine. Not super experience with TS hacks :)

Reply:
Just FYI: the issue seems to be that you cannot use a generic  `assert arg is T` as a method without explicit type annotations: https://github.com/microsoft/TypeScript/issues/34596

Your hack with the exclamation mark disables the type-assertion, `data` after the assertion is still `unknown`.

Other libraries either do not use an asserts type method at all or they are using a asserts function.